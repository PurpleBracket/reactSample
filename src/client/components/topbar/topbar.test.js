/* global describe test expect */
import React from 'react'
import { shallow } from 'enzyme'
import TopbarHome from './topbar.home'
import TopbarDashboard from './topbar.dashboard'

describe('Header', () => {
  test('should render for landing page', () => {
    const wrapper = shallow(<TopbarHome />)
    expect(wrapper).toMatchSnapshot()
  })

  test('should render for app page', () => {
    const wrapper = shallow(<TopbarDashboard />)
    expect(wrapper).toMatchSnapshot()
  })
})
