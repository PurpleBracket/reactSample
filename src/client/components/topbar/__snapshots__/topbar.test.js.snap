// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render for app page 1`] = `
<nav
  aria-label="main navigation"
  className="navbar is-fixed-top is-dark topbarApp"
  role="navigation"
>
  <div
    className="navbar-start"
  >
    <div
      className="navbar-brand"
    >
      <button
        className="button navbar-burger"
        onBlur={[Function]}
        onClick={[Function]}
      >
        <span />
        <span />
        <span />
      </button>
    </div>
  </div>
  <div
    className="navbar-end"
  >
    <UserDropdown
      dark={true}
      transparentBG={true}
    />
  </div>
</nav>
`;

exports[`Header should render for landing page 1`] = `
<nav
  aria-label="main navigation"
  className="navbar is-dark topbarHome is-fixed-top is-transparent"
  role="navigation"
>
  <div
    className="navbar-start"
  >
    <div
      className="navbar-brand"
    >
      <NavLogo
        href="/"
        logoFull={true}
      />
      <button
        className="button navbar-burger"
        onClick={[Function]}
      >
        <span />
        <span />
        <span />
      </button>
    </div>
  </div>
  <div
    className="navbar-end"
  >
    <div
      className="navbar-menu"
    >
      <NavLink
        button={true}
        className="is-primary is-inverted is-outlined"
        href="/signup"
        transparentBG={true}
      >
        Create Account
      </NavLink>
      <NavLink
        button={true}
        className="is-primary is-inverted is-outlined"
        href="/signin"
        transparentBG={true}
      >
        Sign In
      </NavLink>
    </div>
  </div>
</nav>
`;
